apps/screloc/main.cpp

```diff
@@ -270,35 +273,56 @@ class Reloc : public Client::Application {
 						}
 					}
 
+					bool relocated = false;
 					try {
 						org = process(org.get());
+						relocated = true;
 					}
 					catch ( std::exception &e ) {
 						SEISCOMP_ERROR("  + processing failed - %s", e.what());
+						relocated = false;
+					}
+					if ( !org ) { // safety belt, but it should not happen
+						SEISCOMP_ERROR("  + processing failed %s", publicID.c_str());
+						relocated = false;
+					}
+					if ( _ignoreRejected ) {
+						try {
+							if ( org->evaluationStatus() == REJECTED ) {
```

Comment:
`org` can be `nullptr`.

Reply:
This is the only case I need to handle with org being `nullptr`